src/ansiblelint/config.py

```diff
@@ -11,14 +11,14 @@
     {"jinja2": "**/*.j2.*"},
     {"yaml": ".github/**/*.{yaml,yml}"},  # github workflows
     {"text": "**/templates/**/*.*"},  # templates are likely not validable
-    {"execution-environment": "**/execution-environment.yml"},
+    {"execution-environment": "**/execution-environment.{yaml,yml}"},
     {"ansible-lint-config": "**/.ansible-lint"},
-    {"ansible-lint-config": "**/.config/ansible-lint.yml"},
+    {"ansible-lint-config": "**/.config/ansible-lint.{yaml,yml}"},
     {"ansible-navigator-config": "**/ansible-navigator.{yaml,yml}"},
-    {"inventory": "**/inventory/**.yml"},
-    {"requirements": "**/meta/requirements.yml"},  # v1 only
+    {"inventory": "**/inventory/**.{yaml,yml}"},
+    {"requirements": "**/meta/requirements.{yaml,yml}"},  # v1 only
     # https://docs.ansible.com/ansible/latest/dev_guide/collections_galaxy_meta.html
-    {"galaxy": "**/galaxy.yml"},  # Galaxy collection meta
+    {"galaxy": "**/galaxy.{yaml,yml}"},  # Galaxy collection meta
```

Comment:
This is not supported in ansible-core. I prefer `.yaml` but I recently had to switch back to `galaxy.yml` when `ansible-galaxy` couldn't find it.

Reply:
Took me some time to find it in the code: https://github.com/ansible/ansible/blob/4c9385dab720ae5d33054d521cecda31464c8857/lib/ansible/galaxy/dependency_resolution/dataclasses.py#L39

I reverted the change for `galaxy.yml` and also for `execution-environment.yml` where also the `.yaml` extension isn't supported.